jupyter-docker/spark-defaults.conf.template

```diff
@@ -1,13 +1,3 @@
-#######################
-# required by DataProc
-#######################
-
-# Uses the YARN configuration in $HADOOP_CONF_DIR
-
-spark.master yarn
-spark.submit.deployMode client
-spark.executor.instances 10000
-
```

Comment:
this is all done in the dataproc-provided spark-defaults.conf so i'm removing it from here